pallets/vc-management/src/lib.rs

```diff
@@ -165,5 +245,98 @@ pub mod pallet {
 			Self::deposit_event(Event::SomeError { func, error });
 			Ok(Pays::No.into())
 		}
+
+		// Change the schema Admin account
+		#[pallet::weight(195_000_000)]
+		pub fn set_schema_admin(
+			origin: OriginFor<T>,
+			new: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			T::SetAdminOrigin::ensure_origin(origin)?;
+			Self::deposit_event(Event::SchemaAdminChanged {
+				old_admin: Self::schema_admin(),
+				new_admin: Some(new.clone()),
+			});
+			<SchemaAdmin<T>>::put(new);
+			Ok(Pays::No.into())
+		}
+
+		// It requires the schema Admin account
+		#[pallet::weight(195_000_000)]
+		pub fn add_schema(
+			origin: OriginFor<T>,
+			id: Vec<u8>,
+			content: Vec<u8>,
+		) -> DispatchResultWithPostInfo {
+			let sender = ensure_signed(origin)?;
+			ensure!(Some(sender.clone()) == Self::schema_admin(), Error::<T>::RequireSchemaAdmin);
+
+			
+
+			let index = Self::schema_count();
+			<SchemaCount<T>>::put(index + 1);
```

Comment:
Although unlikely, we probably still need to check overflow here, e.g. `checked_add`, maybe `try_mutate` is better too